packages/machine-emulator-tools/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# download machine-emulator tools and create a scratch image the tar.gz contents at root
+FROM busybox as download
+ARG VERSION
+ADD https://github.com/cartesi/machine-emulator-tools/releases/download/v${VERSION}/machine-emulator-tools-v${VERSION}.tar.gz \
+    /opt/cartesi/share/machine-emulator-tools.tar.gz
+RUN tar -C /tmp -xzf /opt/cartesi/share/machine-emulator-tools.tar.gz
```

Comment:
I want `/tmp` to have only the files I'll copy to the next stage (and not the .tar.gz)
Because in the future I'll turn the next stage to only `COPY --from=download /tmp /`
So I can't have the .tar.gz in there.

Reply:
So you coud untar it on a folder inside `/tmp/`, like `/tmp/machine-emulator-tools` and copy from there.

I know it's a temporary stage, but it feels weird to me copying code to `/opt/cartesi`.